src/System.Composition.sln

```diff
@@ -0,0 +1,97 @@
+﻿
```

Comment:
Would you just move the .sln file, or would you move all the System.Composition code under a common \src\System.Composition folder?